docs/_user_guide/data_transfer.md

```diff
@@ -1,339 +1,257 @@
-- On the next prompt, RClone provides you with a unique URL that you need to paste into a browser. After logging in with your Stanford credentials, Google Drive will give a code to paste back into the terminal window.
+7. In Activity Tab, you can see how the transfer is going and if there are any errors. 
 
-![Prompt Map](../images/rclone3.png)
+#### Setting up globus on your laptop 
 
-- Finally, to finish the configuration in the terminal window setup, click "Y" and "enter" to complete the process if you believe everything is properly setup. In the last prompt, hit "q" to quit. You have setup RClone successfully on your Google Drive.
+To setup a data transfer from your personal computer to the yens, first download [globus Connect Personal](https://app.globus.org/file-manager/gcp){:target="_blank"}
+for your operating system and install it on your computer.
 
+If installed correctly, you should see your personal Globus endpoint under Collections in the globus web app.
 
-## RClone Common Operations
-### Initiaitaing a current version of RClone
-``` bash
-ml rclone
-```
-** OR **  
-``` bash
-moudle load rclone
-```
-Note: this needs to be run once per server per terminal session. After you have initiated `module load rclone,` you will only need to call `rclone` as normally from the command line. 
+![Personal Globus Endpoint](/assets/images/personal-globus.png)
 
-### To list connection points
-``` bash
-rclone listremotes
-```
+You can now search for your personal endpoint when setting up a file transfer to the yens.
 
-***Create a remote folder on Google Drive using Rclone. Note this will make the folder within your Google Drive base folder.***
-``` bash
-rclone mkdir smancusoGoogleDrive:GoogleDriveFolderName
-```
+## LFTP
 
-***To upload contents of a directory to Google Drive using copy and Rclone.***
-``` bash
-rclone copy /Path/To/Folder/ smancusoGoogleDrive:GoogleDriveFolderName/
-```
+FTP (File Transfer Protocol) is a standard network protocol used to transfer files between a client and a server over the internet or a local network. People use it to upload, download, or manage files, especially for website maintenance or large data transfers.  Due to security concerns, the Yen servers cannot host a FTP server – you can, however, use an FTP client to connect to an external server that is hosting data. We recommend using lftp, a powerful command-line FTP client that supports advanced features like secure transfers, parallel downloads, resumable transfers, and scripting, making it a preferred choice for efficient and reliable file transfers.
 
-***List contents of remote folder on Google Drive***
-``` bash
-rclone ls smancusoGoogleDrive:GoogleDriveFolderName
-```
-
-***Download from remote Google Drive.***
-``` bash
-rclone copy smancusoGoogleDrive:GoogleDriveFolderName /Path/To/Local/Download/Folder
+### SSH into the Server
```

Comment:
let's just cut all of these examples, they're uncommon and not that instructive.